docs/src/components/Header/Header.js

```diff
@@ -25,14 +25,32 @@ const Header = ({ relativePagePath }) => {
     localStorage.removeItem('leftNavScrollPosition');
   };
 
+  const isMobile = () => {
+    const currURL = typeof window !== 'undefined' && window.location.pathname;
+    if (currURL && currURL.includes('mobile')) {
```

Comment:
Remove the if-else condition with `return currURL && currURL.includes('mobile')`